src/gu/sceGuInit.c

```diff
@@ -12,39 +12,266 @@
+		ZV(LIGHT1_DIRECTION_X),
+		ZV(LIGHT1_DIRECTION_Y),
+		ZV(LIGHT1_DIRECTION_Z),
+		ZV(LIGHT2_DIRECTION_X),
+		ZV(LIGHT2_DIRECTION_Y),
+		ZV(LIGHT2_DIRECTION_Z),
+		ZV(LIGHT3_DIRECTION_X),
+
+		ZV(LIGHT3_DIRECTION_Y),
+		ZV(LIGHT3_DIRECTION_Z),
+		ZV(LIGHT0_CONSTANT_ATTEN),
+		ZV(LIGHT0_LINEAR_ATTEN),
+		ZV(LIGHT0_QUADRATIC_ATTEN),
+		ZV(LIGHT1_CONSTANT_ATTEN),
+		ZV(LIGHT1_LINEAR_ATTEN),
+		ZV(LIGHT1_QUADRATIC_ATTEN),
 
+		ZV(LIGHT2_CONSTANT_ATTEN),
+		ZV(LIGHT2_LINEAR_ATTEN),
+		ZV(LIGHT2_QUADRATIC_ATTEN),
+		ZV(LIGHT3_CONSTANT_ATTEN),
+		ZV(LIGHT3_LINEAR_ATTEN),
+		ZV(LIGHT3_QUADRATIC_ATTEN),
+		ZV(LIGHT0_EXPONENT_ATTEN),
+		ZV(LIGHT1_EXPONENT_ATTEN),
+
+		ZV(LIGHT2_EXPONENT_ATTEN),
+		ZV(LIGHT3_EXPONENT_ATTEN),
+		ZV(LIGHT0_CUTOFF_ATTEN),
+		ZV(LIGHT1_CUTOFF_ATTEN),
+		ZV(LIGHT2_CUTOFF_ATTEN),
+		ZV(LIGHT3_CUTOFF_ATTEN),
+		ZV(LIGHT0_AMBIENT),
+		ZV(LIGHT0_DIFFUSE),
+
+		ZV(LIGHT0_SPECULAR),
+		ZV(LIGHT1_AMBIENT),
+		ZV(LIGHT1_DIFFUSE),
+		ZV(LIGHT1_SPECULAR),
+		ZV(LIGHT2_AMBIENT),
+		ZV(LIGHT2_DIFFUSE),
+		ZV(LIGHT2_SPECULAR),
+		ZV(LIGHT3_AMBIENT),
+
+		ZV(LIGHT3_DIFFUSE),
+		ZV(LIGHT3_SPECULAR),
+		ZV(CULL),
+		ZV(FRAME_BUF_PTR),
+		ZV(FRAME_BUF_WIDTH),
+		ZV(Z_BUF_PTR),
+		ZV(Z_BUF_WIDTH),
+		ZV(TEX_ADDR0),
+
+		ZV(TEX_ADDR1),
+		ZV(TEX_ADDR2),
+		ZV(TEX_ADDR3),
+		ZV(TEX_ADDR4),
+		ZV(TEX_ADDR5),
+		ZV(TEX_ADDR6),
+		ZV(TEX_ADDR7),
+		CV(TEX_BUF_WIDTH0, 512 * 512),
+
+		ZV(TEX_BUF_WIDTH1),
+		ZV(TEX_BUF_WIDTH2),
+		ZV(TEX_BUF_WIDTH3),
+		ZV(TEX_BUF_WIDTH4),
+		ZV(TEX_BUF_WIDTH5),
+		ZV(TEX_BUF_WIDTH6),
+		ZV(TEX_BUF_WIDTH7),
+		ZV(CLUT_ADDR),
+
+		ZV(CLUT_ADDR_UPPER),
+		ZV(TRANSFER_SRC),
+		ZV(TRANSFER_SRC_W),
+		ZV(TRANSFER_DST),
+		ZV(TRANSFER_DST_W),
+		CV(TEX_SIZE0, 257),
+		ZV(TEX_SIZE1),
+		ZV(TEX_SIZE2),
+
+		ZV(TEX_SIZE3),
+		ZV(TEX_SIZE4),
+		ZV(TEX_SIZE5),
+		ZV(TEX_SIZE6),
+		ZV(TEX_SIZE7),
+		ZV(TEX_MAP_MODE),
+		ZV(TEX_SHADE_MAPPING),
+		ZV(TEX_MODE),
+
+		ZV(TEX_FORMAT),
+		ZV(LOAD_CLUT),
+		ZV(CLUT_FORMAT),
+		ZV(TEX_FILTER),
+		ZV(TEX_WRAP),
+		ZV(TEX_LEVEL),
+		ZV(TEX_FUNC),
+		ZV(TEX_ENV_COLOR),
+
+		ZV(TEX_FLUSH),
+		ZV(TEX_SYNC),
+		ZV(FOG1),
+		ZV(FOG2),
+		ZV(FOG_COLOR),
+		ZV(TEX_LOD_SLOPE),
+		ZV(FRAMEBUF_PIX_FORMAT),
+		ZV(CLEAR_MODE),
+
+		ZV(SCISSOR1),
+		ZV(SCISSOR2),
+		ZV(MIN_Z),
+		ZV(MAX_Z),
+		ZV(COLOR_TEST),
+		ZV(COLOR_REF),
+		ZV(COLOR_TESTMASK),
+		ZV(ALPHA_TEST),
+
+		ZV(STENCIL_TEST),
+		ZV(STENCIL_OP),
+		ZV(Z_TEST),
+		ZV(BLEND_MODE),
+		ZV(BLEND_FIXED_A),
+		ZV(BLEND_FIXED_B),
+		ZV(DITH0),
+		ZV(DITH1),
+
+		ZV(DITH2),
+		ZV(DITH3),
+		ZV(LOGIC_OP),
+		ZV(Z_MASK),
+		ZV(MASK_COLOR),
+		ZV(MASK_ALPHA),
+		ZV(TRANSFER_START),
+		ZV(TRANSFER_DST_OFFSET),
+
+		ZV(TRANSFER_SIZE),
+		ZV(TRANSFER_OUT_SIZE),
+		ZV(TRANSFER_OUT_FORMAT),
+		ZV(VERTEX_WEIGHT),
+		ZV(VERTEX_WEIGHT_ADDR),
+		ZV(VERTEX_WEIGHT_SIZE),
+		ZV(VERTEX_WEIGHT_TYPE),
+		ZV(VERTEX_WEIGHT_MODEL),
+
+		ZV(VERTEX_WEIGHT_NORMALIZE),
+		ZV(VERTEX_WEIGHT_OFFSET),
+		ZV(VERTEX_WEIGHT_SCALE),
+
+		ZV(TRANSFER_OUT_SIZE),
+		ZV(TEX_MAP_MODE),
```

Comment:
```suggestion
		ZV(FINISH),
		ZV(END),
```
Like this these match.

Reply:
Done!